src/app/loadout-drawer/subclass-drawer/Plug.tsx

```diff
@@ -0,0 +1,39 @@
+import ClosableContainer from 'app/dim-ui/ClosableContainer';
+import { DefItemIcon } from 'app/inventory/ItemIcon';
+import { DestinyInventoryItemDefinition } from 'bungie-api-ts/destiny2';
+import clsx from 'clsx';
+import React from 'react';
+import styles from './Plug.m.scss';
+
+export default function Plug({
```

Comment:
I'm a bit surprised at how much new code has to be written for this, since we already had mod picker and mod selection.

Reply:
I will see if I can reuse some more. 

We are still heavily reusing the mod picker code, you can see it [here](https://github.com/DestinyItemManager/DIM/pull/7186/files#diff-d9175392bdab55c049c8036f076dc14addc4eadd27713cc81a0241820b9208e8R82-R92). I split the plug drawer out from mod picker specifically for this.

All the new code is for the drawer where you pick your abilities and can open the mod picker style drawer.

Reply:
Looking at `Plug` specifically, that is used by the abilities and to show the currently equipped mods. I think the main reason I didn't reuse selected mod code is that only ever allows for removal of mods, whereas with these we need two types of behaviour. 
- If it is a selected mod, show a close icon for removal.
- If it is an ability we have on click behaviour but no removal. 
- 
This aims at having a single component for this sheet that does both.